README.md

```diff
@@ -153,33 +152,6 @@ If you need additional flexibility, you can use the following building blocks to
 * [OSRM Text Instructions for Swift](https://github.com/Project-OSRM/osrm-text-instructions.swift/) (also compatible with macOS, tvOS, and watchOS)
   * Localized guidance instructions
 
-### Installing Mapbox Core Navigation
```

Comment:
How about moving this content to a separate Markdown file? MapboxNavigation isn’t a one-size-fits-all solution, so we should have something linkable for situations when a developer finds Core Navigation to be more practical.